outlook/src/taskpane/components/Company/CompanySection/CompanySection.tsx

```diff
@@ -247,6 +255,73 @@ class CompanySection extends React.Component<CompanySectionProps, CompanySection
         );
     }
 
+    private getNoCompanyInsightsSection = () => {
+        let NoInsightsSection = null;
+        const company = this.state.company as Company;
+        if (company.isEmpty()) {
+            if (this.props.partner.isAddedToDatabase() && !this.enriched) {
+                NoInsightsSection = (
+                    <div>v
```

Comment:
There is a lost "v" character here, right?

Reply:
Yes right, this has now been corrected.